FirebaseInstallations/Source/Library/Public/FIRInstallations.h

```diff
@@ -14,39 +14,101 @@
 @interface FIRInstallations : NSObject
 
-// TODO: Add `+ (FIRInstallations *)installations;`
+/**
+ * Returns a default instance of @c FIRInstallations.
+ * @return Returns an instance of @c FIRInstallations for @c[FIRApp defaultApp]. Returns @c nil if
+ * the default app is not configured yet.
+ */
++ (nullable FIRInstallations *)installations;
 
+/**
+ * Returns an instance of @c FIRInstallations for an application.
+ * @param application A configured @c FIRApp instance.
+ * @return Returns an instance of @c FIRInstallations corresponding to the passed application.
+ */
 + (FIRInstallations *)installationsWithApp:(FIRApp *)application NS_SWIFT_NAME(installations(app:));
 
+/**
+ * The method creates or retrieves an installation ID. The installation ID is a stable identifier
+ * that uniquely identifies the app instance. NOTE: If the application already has an existing
+ * FirebaseInstanceID then the InstanceID ideintifier will be used.
+ * @param completion A completion handler which is invoked when the operation completes. See @c
+ * FIRInstallationsIDHandler for additional details.
+ */
 - (void)installationIDWithCompletion:(FIRInstallationsIDHandler)completion;
 
+/**
+ * Retrives (locally if exists or from the server) a valid authorization token. An existing token
+ * may be invalidated or expire, so it is recommended the auth token before each server request. The
+ * method does the same as @c-[FIRInstallations authTokenForcingRefresh:completion:] with forcing
+ * refresh @c NO.
+ * @param completion A completion handler which is invoked when the operation completes. See @c
+ * FIRInstallationsTokenHandler for additional details.
+ */
 - (void)authTokenWithCompletion:(FIRInstallationsTokenHandler)completion;
 
+/**
+ * Retrives (locally or from the server depending on @c forceRefresh value) a valid authorization
+ * token. An existing token may be invalidated or expire, so it is recommended the auth token before
+ * each server request. This method should be used with @c forceRefresh == YES when e.g. a request
+ * with the previously fetched auth token failed with "Not Authorized" error.
+ * @param forceRefresh If @c YES then the locally chached auth token will be ignored and a new one
+ * will be requested from the server. If @c NO, then the locally cached auth token will be returned
+ * if exists and has not expired yet.
+ * @param completion  A completion handler which is invoked when the operation completes. See @c
+ * FIRInstallationsTokenHandler for additional details.
+ */
 - (void)authTokenForcingRefresh:(BOOL)forceRefresh
                      completion:(FIRInstallationsTokenHandler)completion;
 
-- (void)deleteWithCompletion:(void (^)(NSError *__nullable))completion;
+/**
+ * Deletes all the installation data including the unique indentifier, auth tokens and
+ * all related data on the server side. The network connection is required for the method to
+ * succeed. If fails, the existing instalation data remains untouched.
```

Comment:
installation

Reply:
I should switch to Xcode 11 with the spellchecker 😄